core/src/main/scala/io/projectglow/transformers/blockvariantsandsamples/VariantSampleBlockMaker.scala

```diff
@@ -27,6 +27,16 @@ import org.apache.spark.sql.types.{ArrayType, IntegerType, StringType}
 
 private[projectglow] object VariantSampleBlockMaker extends GlowLogging {
 
+  def filterStdDevZero(df: DataFrame): DataFrame = {
+    val filteredDf = df.filter(size(array_distinct(col(valuesField.name))) > 1)
+    val numOrig = df.count()
```

Comment:
If we're starting with a VCF or Plink file here, this could be expensive because we'd have to parse the input file 3 times (once to count original, once to count filtered, and once for the downstream computation). I'm not sure it's worth it for this warning.

Reply:
I considered caching, but I wasn't sure if it was worth it. We can always print a warning; it shouldn't be too hard for users to figure out if they lost any variants.